build-tests/localization-plugin-test/build.js

```diff
@@ -0,0 +1,27 @@
+const fsx = require('fs-extra');
```

Comment:
do we want to use fs-extra?

Reply:
No - I'll fix that.

Reply:
Are you gonna use the FileDeletion utility I thought should be extracted to node-core-library?